hoot-core/src/main/cpp/hoot/core/io/OsmApiNetworkRequest.cpp

```diff
@@ -103,10 +104,8 @@ bool OsmApiNetworkRequest::networkRequest(QUrl url, QNetworkAccessManager::Opera
   //  Check error status on our reply
   if (QNetworkReply::NoError != reply->error())
   {
-    QString errMsg = reply->errorString();
-    throw HootException(QString("Network error for request (%1): %2\n%3")
-      .arg(url.toString())
-      .arg(errMsg).arg(QString(_content)));
+    _error = reply->errorString();
+    return false;
```

Comment:
Keeping the error string and returning `false` is a better way to handle this error.